cgmanifest.json

```diff
@@ -309,6 +309,69 @@
             }
          }
       },
+      {
+         "component": {
+             "git": {
+                 "commitHash": "02a2a458ac15912d7d87cc1171e811b0c5219ece",
+                 "repositoryUrl": "https://github.com/grpc/grpc"
+             },
+             "type": "git"
+         }
+      },
+      {
+         "component": {
+             "git": {
+                 "commitHash": "cc4bed2d74f7c8717e31f9579214ab52a9c9c610",
+                 "repositoryUrl": "https://github.com/abseil/abseil-cpp"
+             },
+             "type": "git"
+         }
+     },
+     {
+         "component": {
+             "git": {
+                 "commitHash": "b29b21a81b32ec273f118f589f46d56ad3332420",
+                 "repositoryUrl": "https://github.com/google/boringssl.git"
+             },
+             "type": "git"
+         }
+     },
+     {
+         "component": {
+             "git": {
+                 "commitHash": "3be1924221e1326df520f8498d704a5c4c8d0cce",
+                 "repositoryUrl": "https://github.com/c-ares/c-ares.git"
+             },
+             "type": "git"
+         }
+     },
+     {
+         "component": {
+             "git": {
+                 "commitHash": "6599cac0965be8e5a835ab7a5684bbef033d5ad0",
+                 "repositoryUrl": "https://github.com/llvm-mirror/libcxx.git"
+             },
+             "type": "git"
+         }
+     },
+     {
+         "component": {
+             "git": {
+                 "commitHash": "9245d481eb3e890f708ff2d7dadf2a10c04748ba",
+                 "repositoryUrl": "https://github.com/llvm-mirror/libcxxabi.git"
+             },
+             "type": "git"
+         }
+     },
+     {
+         "component": {
+             "git": {
+                 "commitHash": "9ce4a77f61c134bbed28bfd5be5cd7dc0e80f5e3",
+                 "repositoryUrl": "https://github.com/google/upb.git"
+             },
+             "type": "git"
+         }
+     },
```

Comment:
Please add golang here, if it is needed for building onnxruntime server.

Reply:
My understanding based on discussions with CELA was that we don't need to include it. The specific instruction was 
```
If the dependency is installed by a user by running the instructions or script and isn’t build by Microsoft, I don’t consider it as a distribution that Microsoft has to track. 
```
I will add you to the email thread.